challenges/fizzBuzzTree/fizz-buzz-tree.js

```diff
@@ -0,0 +1,16 @@
+const fizzBuzzTree = (tree) => {
+  if(tree.value % 3 === 0 && tree.value % 5 === 0) {
```

Comment:
to make this a little shorter, you can just check if the value is divisible by 15